### DISCUSSION TITLE
Experimental/Beta/Development features

### DISCUSSION THREAD

**Discussion by Daeraxa** [pulsar-edit/.github#149] [Core Application & Packages]
As was discussed on the search settings PR, we have to have a way to evaluate new and in-development features but not necessarily ship the feature by default.

What about an "Experimental/Beta/Dev" section to the settings-view package that might entice people to look at the features where additional info could be provided as to what it is and how to provide feedback on it?

That way we dont bury it in the normal config settings and have a relatively easy way to link to it from the changelog or welcome package etc.

**Comment by confused-Techie** [comment]
I personally would be a huge fan of implementing something like this.

We might already be on the same page with this, but waned to touch on it, I don't think the settings themselves though should be scoped to this settings page. For example having a new settings object like `experimental` with all of the settings on it seems like a bad idea.

Since I'd hate for once the object comes out of beta for people's settings to get lost. Or if modified get left cluttering up the configuration file for Pulsar.

I'd think the best implementation could be that page in settings view can have hardcoded settings that it pulls out manually to add to this page. 

Pages like the "URI Handler" already do this, so we could model the behavior after that, since iirc the Enable/Disable for URI handling is actually in the `core` settings object rather than anywhere else.

**Reply by Daeraxa** [reply]
> We might already be on the same page with this, but waned to touch on it, I don't think the settings themselves though should be scoped to this settings page. For example having a new settings object like `experimental` with all of the settings on it seems like a bad idea.

Yeah exactly what I meant, the setting might still be part of `core` or `editor` but it would be displayed on the new pane.
